packages/yoga/src/Chips/web/Chips.jsx

```diff
@@ -173,7 +156,11 @@ const Chips = React.forwardRef(
             }}
           />
         )}
-        {children && <Text>{children}</Text>}
+        {children && (selected || disabled) ? (
```

Comment:
Shouldn't we display the `Text.Caption` only when the `children` exist to maintain consistency with previous behavior?

Reply:
Yes, that's what happens in this piece of code, right?

Reply:
The `Text.Caption` will be rendered when the condition is `false`.
![Screenshot 2024-03-19 at 08 51 05](https://github.com/gympass/yoga/assets/141146776/3d7d7346-728b-4275-99f3-54439f7f7715)

We could create a variable to define the text component.
```jsx
const TextComponent = selected || disabled ? Text.Overline : Text.Caption;

children && <TextComponent>{children}</TextComponent>
```

Reply:
You're right, I'll work on the change!